src/loaders/lottie/tvgLottieModel.h

```diff
@@ -665,16 +665,16 @@ struct LottieGradientStroke : LottieGradient, LottieStroke
 
 struct LottieImage : LottieObject, LottieRenderPooler<tvg::Picture>
 {
-    union {
-        char* b64Data = nullptr;
-        char* path;
-    };
-    char* mimeType = nullptr;
-    uint32_t size = 0;
-    float width = 0.0f;
-    float height = 0.0f;
+    LottieImageData data;
```

Comment:
Suggestion: `BitmapData bitmap`

Reply:
I see every LottieProperty has `Lottie` prefix in their name. I renamed with `LottieBitmap`, please have a look